src/pages/docs/approvals/servicenow/index.md

```diff
@@ -217,7 +217,7 @@ e.g `Octopus: Deploy "Web Site" version 1.0.1-hotfix-001 to "Dev"`
 The title must match the format **exactly**, including the double-quotes.
 :::
 
-### Populating CR fields through Octopus
+### Populating CR fields through Octopus (Available from 2024.2 on)
```

Comment:
Can we make this similar to the below:

<img width="851" alt="image" src="https://github.com/OctopusDeploy/docs/assets/1035315/af3026f8-c625-461c-be93-cc8fe8ac878b">